### DISCUSSION TITLE
Hi, Need Help for a Moti & React Native CLI with TypeScript

### DISCUSSION THREAD

**Discussion by samsolo-droid** [nandorojo/moti#206] [Q&A] [answered]
Hello @nandorojo, thank you for this amazing library you made for simplifying our way for animation in RN. I'm getting this error  "ERROR  TypeError: null is not an object (evaluating 'dispatcher.useContext')

This error is located at:
    in Moti (at App.tsx:16)
    in FadeIn (at App.tsx:27)
    in RCTView (at View.js:32)
    in View (at App.tsx:25)
    in App (at renderApplication.js:50)
    in RCTView (at View.js:32)
    in View (at AppContainer.js:92)
    in RCTView (at View.js:32)
    in View (at AppContainer.js:119)
    in AppContainer (at renderApplication.js:43)
    in MobileVPNApp(RootComponent) (at renderApplication.js:60)"
for all the times I wanted to integrate Moti in RN with Typescript. But Everything works fine when it's without TypeScript. I'm not using Expo. But I love using TypeScript in all my JS project.

Can you help me fix it?

Excuse me if I made mistakes, I'm Togolese so I'm just trying English. Thank you

**Comment by nandorojo** [comment, marked as answer]
This tends to happen if you have multiple versions of react installed https://github.com/nandorojo/moti/issues/189

If you're using npm, try `npm i --legacy-peer-deps`

**Reply by samsolo-droid** [reply]
OK thanks
